policies/rudderc/docs/src/modules/template.md

```diff
@@ -0,0 +1,477 @@
+# Template module
+
+The module supports the following template engines:
+
+* MiniJinja (which is a subset of Jinja2)
+* Mustache
+* Jinja2 (available only on Linux)
+
+Minijinja is the default and preferred engine, Jinja2 and Mustache are provided for backward compatibility.
```

Comment:
mWe could explain more here: MiniJinja and Mustache are native and fast, Jinja2 is very slow, requires Python but allows dynamic extensions.